modules/http-api/src/main/java/org/mule/runtime/http/api/domain/message/HttpMessage.java

```diff
@@ -48,6 +49,11 @@
    */
   Collection<String> getHeaderValuesIgnoreCase(String headerName);
 
+  /**
+   * @return all headers
+   */
+  MultiMap<String, String> getHeaders();
```

Comment:
Is this mutable or immutable? Maybe good idea to be clear about this in JDoc too.